src/components/InputDropdown/index.js

```diff
@@ -18,21 +18,23 @@ class InputDropdown extends React.Component {
   }
 
   componentDidMount() {
-    window.addEventListener('mousedown', (e) => {
-      this.setState((prevState) => {
-        // clicked outside list
-        if (
-          (prevState.isOpen && !this.listEl.contains(e.target)) ||
-          e.target === this.inputEl
-        ) {
-          return { isOpen: !this.state.isOpen }
-        }
-      })
-    })
+    window.addEventListener('mousedown', this.handleClick)
   }
 
   componentWillUnmount() {
-    window.removeEventListener('mousedown')
```

Comment:
I stumbled across this error while testing the back button (no function passed to the `removeEventListener`.) Figured I'd fix it while I was in here.

Reply:
👍